submissions/tamara_tumeh/assignment8.js

```diff
@@ -0,0 +1,37 @@
+// T1
+const editor = {
+    title: 'my first blog',
+    getUpperTitle() {
+        return this.title.toUpperCase();
+    }
+};
+const getTitle = editor.getUpperTitle.bind(editor);
+console.log(getTitle());
+///////////////////////////////////////////////////////////////
+// T2
+const formHandler = {
+    value: 'initial',
+    onChange(newValue) {
+        this.value = newValue;
+    }
+};
+function simulateInputChange(callback) {
+    callback('updated');
+}
+simulateInputChange(formHandler.onChange.bind(formHandler));
```

Comment:
could u please use arrow function as another solution here? and explain why it could work